packages/icon/src/Icon.tsx

```diff
@@ -34,12 +35,12 @@ export const createIcon = (children: ReactNode, name: string) => {
 		'aria-hidden': ariaHidden = 'true',
 		'aria-label': ariaLabel,
 		className,
-		size = 'md',
 		color,
-		weight = 'regular',
+		size = 'md',
 		style,
+		weight = 'regular',
```

Comment:
Just re-ordering the props here to be alphabetical (and match the order in the typescript type)